views/scss/qti/_qti-base.scss

```diff
@@ -11,7 +11,7 @@ $mainContainer : '.qti-item';
 @import "inc/mediaelementplayer";
 
 %selectableHover {
-    background: whiten($info, .95) !important;
+    background: whiten($info, .9) !important;
```

Comment:
**nitpick:** This will introduce regressions as it will impact any theme.
The issue you try to fix is on a customer-specific theme. Please first lookt at it, and see how you can align the colors between the keyboard and mouse focus styles.